Jenkinsfile.deploy

```diff
@@ -56,6 +56,10 @@ pipeline {
         // Bot author information. Set as params for easy testing.
         string(name: 'GIT_AUTHOR_BOT', defaultValue: 'bsig-gh-bot', description: 'From which author should the PR be created ?')
         string(name: 'BOT_CREDENTIALS_ID', defaultValue: 'bsig-gh-bot', description: 'Credentials for PR creation')
+
+        //Build branch name for quickstarts
+        string(name: 'QUICKSTARTS_BUILD_BRANCH_NAME', defaultValue: 'development', description: 'Base branch for quickstarts. Set if you are not on a multibranch pipeline.')
```

Comment:
This is only in to checkout `development` instead of `master` when `BUILD_BRANCH_NAME=master` or ?
when I specify another branch in `BUILD_BRANCH_NAME`, then `QUICKSTARTS_BUILD_BRANCH_NAME` will be the same or ?